src/js/formprint/index.js

```diff
@@ -0,0 +1,98 @@
+import 'js/base/setup';
+import 'js/i18n';
```

Comment:
Is this needed?

Reply:
`import 'js/i18n';` doesn't seem to be needed. Removed it without issue.

Removing `import 'js/base/setup';` breaks the page and throws this browser console error:

```
Uncaught TypeError: dayjs__WEBPACK_IMPORTED_MODULE_1___default(...).utc is not a function
```

Looks like we need the `dayjs` global window variable that is set in the `js/base/setup` file: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/base/setup.js#L43

Replacing the `import 'js/base/setup` import with this fixes the error:

```js
import dayjs from 'dayjs';
import '../base/dayjs';

window.dayjs = dayjs;
```

But it is messier.